chaco/legend.py

```diff
@@ -358,7 +355,7 @@ def get_preferred_size(self):
         if len(self.plots) == 0:
             return [0, 0]
 
-        plot_names, visible_plots = list(sm.map(list, sm.zip(*sorted(self.plots.items()))))
+        plot_names, visible_plots = list(map(list, zip(*sorted(self.plots.items()))))
```

Comment:
I'm not a big fan of such code - `list(map(list, zip(*sorted(self.plots.items()))))` - because i find it hard to understand. Anyway, that's not an actionable item in this PR given the lack of context.

Reply:
I agree, any code with "`)))))`" smells a bit, but that is a problem for another day/PR